Source/Instrument Addon.lvproj

```diff
@@ -947,22 +931,22 @@
 				<Property Name="Bld_excludeLibraryItems" Type="Bool">true</Property>
 				<Property Name="Bld_excludePolymorphicVIs" Type="Bool">true</Property>
 				<Property Name="Bld_excludeTypedefs" Type="Bool">true</Property>
-				<Property Name="Bld_localDestDir" Type="Path">../Built/Display Templates</Property>
+				<Property Name="Bld_localDestDir" Type="Path">../Built/Custom Devices/Display Templates</Property>

```

Comment:
This and all Display Templates related Build assets need to go into "/Built/Display Templates" like the Instrument Tool.